lib/mocha.js

```diff
@@ -145,14 +145,24 @@ Mocha.prototype.reporter = function(reporter, reporterOptions) {
     if (reporters[reporter]) {
       _reporter = reporters[reporter];
     }
-    // Try to load reporters from process.cwd() and node_modules
+    // Try to load reporters from process.cwd()
+    if (!_reporter) {
+      try {
+        _reporter = require(path.resolve(reporter));
```

Comment:
This is going to be a problem if we're running in a browser.  Why did this need to move out of `_mocha`?